### DISCUSSION TITLE
Add Camera Name to Debug Log Line

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#12798] [Ideas]
Per this discussion and troubleshooting a camera audio event, it would be nice if the camera name was in the log line so one could filter the camera in question. Currently the only work around is to disable audio on all the other cameras to debug the one having trouble which is cumberson and not ideal:

https://github.com/blakeblackshear/frigate/discussions/12797#discussioncomment-10258217

Something like:

```

2024-08-06 21:40:49 frigate.events.audio - camera:<camera_name> --- Heard mechanisms with a score of 0.66796875
```

**Comment by NickM-27** [comment]
Implemented in https://github.com/blakeblackshear/frigate/pull/12799
